Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM python:3.6-slim
+
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update -qq && \
+  apt-get install -y --no-install-recommends \
+  build-essential \
+  wget \
+  openssh-client \
+  graphviz-dev \
+  pkg-config \
+  git-core \
+  openssl \
+  libssl-dev \
+  libffi6 \
+  libffi-dev \
+  libpng-dev \
+  curl && \
+  apt-get clean && \
+  rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
+  mkdir /app
+
+WORKDIR /app
+
+# Copy as early as possible so we can cache ...
+COPY requirements.txt .
+
+RUN pip install -r requirements.txt --no-cache-dir
+
+COPY . .
+
+RUN pip install -e . --no-cache-dir
+
+VOLUME ["/app/model", "/app/config", "/app/project"]
+
+EXPOSE 5005
+
+ENTRYPOINT ["./entrypoint.sh"]
+
+CMD ["start", "-d", "./dialogue"]
```

Comment:
does this path name still make sense? should be `./models` or something no?

Reply:
these dockerfiles need some love - they need to use the rasa cli tobias created instead of the arbitrary commands. I'd like to do that separately though

Reply:
I've created a separate issue for that https://github.com/RasaHQ/rasa_nlu/issues/1795